packages/genesys-spark-components/src/components/stable/gux-pagination-cursor/a11yManualChecklist.md

```diff
@@ -12,5 +12,5 @@
 | ✅ | [3.2.2 On Input](https://www.w3.org/WAI/WCAG21/Understanding/on-input.html) | - |
 | ✅ | [3.3.1 Error Identification](https://www.w3.org/WAI/WCAG21/Understanding/error-identification.html) | - |
 | ✅ | [3.2.2 Labels or Instructions](https://www.w3.org/WAI/WCAG21/Understanding/labels-or-instructions.html) | - |
-| ❌ | [4.1.2: Name, Role, Value](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html) | COMUI-799 the 'next' and 'previous' buttons should convey which content it is associated with |
+| ✅ | [4.1.2: Name, Role, Value](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html) | - |
```

Comment:
Should the links be pointing to WCAG 2.2? Looks like they still point to 2.1

Reply:
Rebase off main will fix this 👍